.gitignore

```diff
@@ -20,4 +20,4 @@ yarn-error.log*
 
 
 #TODO add .env to your gitignore
-
+server/.env
```

Comment:
It doesn't look like this worked as your env file is still appearing in the diff, did you git remove the file in addition to adding it here?

Reply:
cleared cache. gitignore is now fixed